src/nvim/ex_cmds2.c

```diff
@@ -2347,11 +2363,12 @@ int do_in_runtimepath(char_u *name, int all, DoInRuntimepathCB callback,
 
           // Expand wildcards, invoke the callback for each match.
           if (gen_expand_wildcards(1, &buf, &num_files, &files,
-                                   EW_FILE) == OK) {
+                                   (flags & DIP_DIR) ? EW_DIR
+                                                     : EW_FILE) == OK) {
```

Comment:
If I need to break ternary like this, I break it into three lines, not two. Though in this case it is better to move `== OK` on next line, not part of the ternary.